docs/pages/access-controls/guides/moderated-sessions.mdx

```diff
@@ -65,27 +65,40 @@ The following fields are optional for `require_session_join`:
 |---|---|---|
 |`on_leave`|[On leave](#on-leave)|The action to take when the policy is no longer satisfied|
 
-#### Example
+#### Example Session Moderation Required Role
```

Comment:
This reads like a proper noun, but we don't really have a first class resource called "Session Moderation Required Role".

I would personally just keep `Example` here.

Reply:
Or the confusion here may be because we don't use title case in our titles, for whatever reason.

Reply:
This is to make it clearer which terms in a heading are proper nouns